m/mint/mint_ubi_8.4.sh

```diff
@@ -0,0 +1,57 @@
+# ----------------------------------------------------------------------------
+#
+# Package        : mint
+# Version        : 
+# Source repo    : https://github.com/bifurcation/mint
+# Tested on      : UBI 8.4
+# Script License : Apache License, Version 2 or later
+# Maintainer     : Sapana Khemkar <spana.khemkar@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+
+#!/bin/bash
+
+# set -u
+PACKAGE_NAME=mint
+PACKAGE_VERSION="$v0.0.0-20180715133206-93c51c6ce115"
```

Comment:
Can you check whether we can use commit id as PACKAGE_VERSION.
Also use something like :
`PACKAGE_VERSION=${1:-"v0.0.0-20180715133206-93c51c6ce115"}`

This will take input from command like argument, if nothing specified will take default value as "v0.0.0-20180715133206-93c51c6ce115"

Reply:
As version provided by customer was v0.0.0-20180715133206-93c51c6ce115, I preferred this way.